lib/link_expansion.rb

```diff
@@ -5,11 +5,14 @@
 # The concept is documented in /doc/link-expansion.md
 #
 class LinkExpansion
-  attr_reader :content_id, :locale, :with_drafts
+  attr_reader :content_id, :locale, :edition, :with_drafts
 
-  def initialize(content_id, locale: nil, with_drafts: false)
-    @content_id = content_id
-    @locale = locale || Edition::DEFAULT_LOCALE
+  def initialize(content_id: nil, locale: nil, edition: nil, with_drafts: false)
+    raise "Only (content_id, locale) or edition should be set." if ((content_id || locale) && edition) || (!(content_id || locale) && !edition)
```

Comment:
It's used elsewhere in the app without an edition, for the `GetExpandedLinks` query.